storm-core/src/clj/org/apache/storm/daemon/common.clj

```diff
@@ -42,17 +42,17 @@
 
 (defmacro defserverfn [name & body]
```

Comment:
@abhishekagarwal87 
I didn't intend but it's better to fix it. I'll address it.